flytekit/bin/entrypoint.py

```diff
@@ -80,9 +92,16 @@ def _execute_task(task_module, task_name, inputs, output_prefix, raw_output_data
 
                 _data_proxy.Data.get_data(inputs, local_inputs_file)
                 input_proto = _utils.load_proto_from_file(_literals_pb2.LiteralMap, local_inputs_file)
+                _distributed_training_context = _run_with_retries(
```

Comment:
It will only be passed into the ExecutionParameters of CustomTrainingJobTasks. The reason why we need it in the EngineContext object is that we need to feed this information to `execute()` function so that the `output_persist_predicate` callable can take this information and decide whether or not it wants to persist the output of the task

Reply:
This is fixed